assignment/scripts/aboutMe.js

```diff
@@ -1,59 +1,91 @@
 // REQUIRED FEATURES:
 // 1 - Create a variable called `firstName` and assign it the value of your first name
-
+// used const for name and number since those do not change;
+const firstName = "Michael";
 // 2 - Create a second variable called `lastName` and assign it the value of your last name
-
+const lastName = "Duren";
 // 3 - Create a third variable called `fullName`, assign it the value of your first and last name
 // (remember, you already have variables for this, can you use those?)
-
+const fullName = `${firstName} ${lastName}`;
 // 4 - Console log the value of `fullName`
-
+console.log(fullName);
 // 5 - Create a variable called `luckyNumber` and assign it the value of your lucky number.
-
-// 6 - Console log this sentence, adding in the variables you created above: 
+const luckyNumber = 8;
+// 6 - Console log this sentence, adding in the variables you created above:
 // 'My name is (full name), and I think (lucky number) is a winner!'.
 // Refer back to the videos if you need help with this one.
-
+console.log(`My name is ${fullName}, and I think ${luckyNumber} is a winner!`);
 // 7 - Create a variable named `adventurous` and set it to a boolean value (true or false)
-
+let adventures = true;
```

Comment:
Question 7 specifies a variable name "adventurous", but as long as you are consistent with this naming I think you are fine.